code/modules/clothing/spacesuits/miscellaneous.dm

```diff
@@ -128,8 +128,8 @@ Contains:
 
 	//Space pirate outfit
 /obj/item/clothing/head/helmet/space/pirate
-	name = "pirate hat"
-	desc = "Yarr."
+	name = "royal tricorne"
+	desc = "A tricorne form the Honorable Space Queen herself, that is space proof as well as made with padded armor."

```

Comment:
A thick, space-proof tricorne from the royal Space Queen. It's lined with a layer of reflective kevlar.